js/tc39/breaking_test_errors.json

```diff
@@ -2284,19 +2277,19 @@
   "test/language/expressions/object/method-definition/escaped-set-t.js-strict:true": "test/language/expressions/object/method-definition/escaped-set-t.js: error is not an object (Test262: This statement should not be evaluated.)",
   "test/language/expressions/object/method-definition/escaped-set.js-strict:true": "test/language/expressions/object/method-definition/escaped-set.js: error is not an object (Test262: This statement should not be evaluated.)",
   "test/language/expressions/object/method-definition/fn-name-fn.js-strict:true": "test/language/expressions/object/method-definition/fn-name-fn.js: TypeError: Cannot read property 'enumerable' of undefined at verifyNotEnumerable (harness/propertyHelper.js:212:11(8))",
+  "test/language/expressions/object/method-definition/forbidden-ext/b1/meth-forbidden-ext-direct-access-prop-arguments.js-strict:false": "test/language/expressions/object/method-definition/forbidden-ext/b1/meth-forbidden-ext-direct-access-prop-arguments.js: Test262Error: Expected a TypeError to be thrown but no exception was thrown at all at harness/sta.js:22:9(49)",
+  "test/language/expressions/object/method-definition/forbidden-ext/b1/meth-forbidden-ext-direct-access-prop-caller.js-strict:false": "test/language/expressions/object/method-definition/forbidden-ext/b1/meth-forbidden-ext-direct-access-prop-caller.js: Test262Error: Expected a TypeError to be thrown but no exception was thrown at all at harness/sta.js:22:9(49)",
```

Comment:
methods should not have access to `arguments` and `caller` - seems insignificant for k6.

Reply:
Hmm I don't understand this - [`arguments`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) is not not recommended, but it's not deprecated yet either, and we don't support ["rest parameters"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) yet, right? What am I missing?

Reply:
The [tests](https://github.com/tc39/test262/blob/main/test/language/expressions/object/method-definition/forbidden-ext/b1/meth-forbidden-ext-direct-access-prop-caller.js) test that *methods* do *not* have `arguments` or `caller`.  So in this case they now do ... as goja does not have methods it has only functions ;)